data/remnant missions.txt

```diff
@@ -1713,3 +1717,150 @@ mission "Remnant: Broken Jump Drive job"
+	to offer
+		has "Remnant: Face to Maw 1: done"
+	source "Aventine"
+	destination "Viminal"
+	cargo "slime sample" 1
+	on offer
+		conversation
+			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though."`
+			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
+			choice
+				`	"Yes, I can."`
+					accept
+				`	"I would rather not."`
+			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`
+			decline
+	on accept
+		event "remnant: penguin"
+	on complete
+		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
+		"remnant taely" ++
+		payment 1000000
+		conversation
+			branch taely
+				"remnant taely" > 1
+			`When you land on Viminal a solidly-built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Taely, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`
+			goto main
+			label taely
+			`	When you land on Viminal you spot Taely waiting for you. "Greetings, Captain <first>. It is good to see you again."`
+			label main
+			`	"So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff," she chants. "Let's go back to my shop." She leads the way through several side passages to an out-of-the-way landing pad connected to a large hanger. Inside, you can see a giant tank filled with a murky fluid, surrounded by equipment and displays. She makes her way over to a large machine with several slots which she slides the container into. As the machine starts to work, she turns back to you.`
```

Comment:
Again, there's an issue here where Taely continues from the first paragraph into this one, and so there should not be closing quotation marks. Here are two options: We can remove the ending quotation marks from L1807 and L1810, or we can add an introduction to L1812, such as, "Taely straightens up and says, 'So, Plume sent us...'"

["Hanger" should be replaced with "hangar,"](https://www.merriam-webster.com/words-at-play/hangar-vs-hanger) assuming it's referring to a... hangar. (Which sounds like it's the case, since there's also a landing pad.)

I think we should move the last sentence to the next paragraph, so that we don't start the next one with a huge dialogue textwall.

Reply:
Added `She makes a gesture you are starting to recognize as a sign of curiosity. ` to the start of that line to break up the quotation marks.

Fixed the spelling. 

Moved the sentence.